apis/monitoring-apis/cloud-cost-api.md

```diff
@@ -31,85 +27,76 @@ Refers to the number of line items you are offsetting. Pairs with `limit`. See t
 ```json
 {
     "code": 200,
     "data": {
-        "graphData": [
+        "sets": [
             {
-                "start": "",
-                "end": "",
-                "items": []
+                "cloudCosts": {
+                    "": {
+                        "properties": {
+                            "provider": ""
+                        },
+                        "window": {
+                            "start": "",
+                            "end": ""
+                        },
+                        "listCost": {
+                            "cost": ,
+                            "kubernetesPercent": 
+                        },
+                        "netCost": {
+                            "cost": ,
+                            "kubernetesPercent": 
+                        },
+                        "amortizedNetCost": {
+                            "cost": ,
+                            "kubernetesPercent":
+                        },
+                        "invoicedCost": {
+                            "cost": ,
+                            "kubernetesPercent": 
+                        },
+                        "amortizedCost": {
+                            "cost": ,
+                            "kubernetesPercent": 
+                        }
+                    },
+                },
+                "window": {
+                    "start": "",
+                    "end": ""
+                },
             }
         ],
-        "tableTotal": {
-            "name": "",
-            "kubernetesPercent": 0,
-            "cost": 0
-        },
-        "tableRows": []
+        "window": {
+            "start": "",
+            "end": ""
+        }
     }
 }
 ```
 {% endswagger-response %}
 {% endswagger %}
 
-## Using the `CostMetric` parameter
-
-`CostMetric` values are based on and calculated following standard FinOps dimensions and metrics. The four available metrics supported by the Cloud Cost API are:
-
-| CostMetric value | Description                                                                                 |
-| ---------------- | ------------------------------------------------------------------------------------------- |
-| NetCost          | Costs inclusive of discounts and credits. Will also include one-time and recurring charges. |
-| AmortizedNetCost | `NetCost` with removed cash upfront fees and amortized                                      |
-| ListCost         | CSP pricing without any discounts                                                           |
-| InvoicedCost     | Pricing based on usage during billing period                                                |
-
-Providing a value for `CostMetric` is optional, but it will default to `AmortizedNetCost` if not otherwise provided.
-
-See our [Cloud Cost Metrics](/using-kubecost/navigating-the-kubecost-ui/cloud-costs-explorer/cloud-cost-metrics.md) doc to learn more about these cost metric types and how they are calculated.
```

Comment:
maybe keep this link with a change of text

Reply:
This section exists almost verbatim in the Cloud Cost Explorer doc for the UI, which I think is where it will be more relevant.